### PR TITLE
Adjustments for core notification and icon changes

### PR DESCRIPTION
**Pull request by bx80**
### Description:

This PR updates the UI test screenshots affected by the notification and icon updates in core PR [#21077](https://github.com/matomo-org/matomo/pull/21077)

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: tests/UI/expected-ui-screenshots/CustomAlerts_edit.png
Comment (michalkleiner) [inline tests/UI/expected-ui-screenshots/CustomAlerts_edit.png:1]: Something changed the text colour in the info boxes to a lighter one, does it still pass contrast test?

Comment (michalkleiner) [reply tests/UI/expected-ui-screenshots/CustomAlerts_edit.png:1]: Disregard that, it's actually darker on the new screen, so better contrast. Misunderstood the Github overlay feature.

File: tests/UI/expected-ui-screenshots/CustomAlerts_list.png
Comment (michalkleiner) [inline tests/UI/expected-ui-screenshots/CustomAlerts_list.png:1]: The icon with the plus in the circle isn't centred. Looking at the replacement set in JIRA it should be same size and similarly centred as the previous plus icon.

Comment (bx80) [reply tests/UI/expected-ui-screenshots/CustomAlerts_list.png:1]: Icon sizes have been scaled up to match the previous size, so this should now be centered.

Comment (michalkleiner) [reply tests/UI/expected-ui-screenshots/CustomAlerts_list.png:1]: I still see it slightly off, but it probably looks better than what it was initially.

**Pull request merged**